addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -100,12 +104,20 @@ export class ColorPlugin extends Plugin {
         const hasGradient = isColorGradient(backgroundImage);
         const hasTextGradientClass = el.classList.contains("text-gradient");
 
+        let backgroundColor = elStyle.backgroundColor;
+        const activeTab = document.querySelector(".o_font_color_selector button.active")?.innerHTML.trim();
+        if (backgroundColor.startsWith("rgba") && activeTab === "Solid") {
+            const values = backgroundColor.match(RGBA_REGEX) || [];
+            const alpha = parseFloat(values.pop()); // Extract alpha value
+            if (alpha === RGBA_OPACITY) {
+                backgroundColor = `rgb(${values.slice(0, 3).join(", ")})`; // Remove alpha
+            }
+        }
```

Comment:
Add a comment explaining there is no alpha value in solid colors in the colorpicker but the alpha value is automatically added when the color is selected and therefore the process needs to be reversed here. In fact, here is the comment from the same change in `web_editor`:
```js
            // Buttons in the Solid colors tab of the palette have no
            // opacity, hence they should be searched by removing
            // opacity of 0.6 (which was applied by default) from
            // the selected color.
```